### ISSUE TITLE
idna 0.2 build is broken (presumably by stringprep-1.0.0)

### ISSUE THREAD

**Issue by peti** [Porges/idna-hs#1]
The build fails with the following error message:

```
Text/IDNA.hs:7:8:
    Could not find module `Text.NamePrep'
    Use -v to see a list of the files searched for.
```

A complete log is at http://hydra.cryp.to/build/33739/nixlog/1/raw.

**Comment by Porges** [comment]
Yes, there was no upper bound on the stringprep dependency.

I've fixed this and pushed v0.3.0 to Hackage.

**Comment by Porges** [comment]
If you want to continue to use 0.2 you can install stringprep-0.1.5 (but 1.0.0 is recommended as it is much faster).
